src/docs/product/dashboards/widget-builder/index.mdx

```diff
@@ -62,20 +62,33 @@ In the dataset selection step, choose which type of data you would like to use i
 
 ### Errors & Transactions
 
-Choosing "Errors and Transactions" allows you to query and aggregate error and transaction events in the same way you would for a [Discover Query](/product/discover-queries/). Some widget examples include: 
+Choosing "Errors and Transactions" allows you to query and aggregate error and transaction events in the same way you would for a [Discover Query](/product/discover-queries/). Some widget examples include:
 
-- Tracking performance of an endpoint 
-- Throughput by country 
+- Tracking performance of an endpoint
+- Throughput by country
 - Users most affected by errors
 
 ### Issues
-Choosing "Issues (States, Assignment, Time, etc.)" allows you to query issue properties, such as `is:unresolved` or event properties such as `error.handled:0`, and to sort by issue fields such as `First Seen`. When searching on event properties, the search will return any issue that has one or more events matching the supplied event filters. 
+Choosing "Issues (States, Assignment, Time, etc.)" allows you to query issue properties, such as `is:unresolved` or event properties such as `error.handled:0`, and to sort by issue fields such as `First Seen`. When searching on event properties, the search will return any issue that has one or more events matching the supplied event filters.
 Choose this dataset if you want to customize a list of issues on your dashboard. Some widget examples include:
 
-- Issues assigned to your team 
```

Comment:
I think these changes are because of my linter?

Reply:
yep